src/components/Toast/Toast.Overview.stories.mdx

```diff
@@ -0,0 +1,290 @@
+      };
+      return (
+        <>
+          <Box childGap="md" direction="row">
+            <Button onClick={handleClick}>
+              Open toast with custom duration
+            </Button>
+          </Box>
+        </>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Dismissible (or lack thereof)
+
+By default, all Toasts are dismissible since they
+should not hinder the user experience, but we still wish to provide a quick and easy
+escape hatch, should a user want to clear it out of their view.
+
+In cases where you'd like the toast to remain for a set amount of time
+without the user being able to clear it, you can use `isDismissable: false`.
+
+<Canvas isExpanded>
+  <Story name="Dismissible">
+    {() => {
+      const handleClick = () => {
+        toast(`this toast can't be dismissed`, { isDismissable: false, duration: 100000 });
+      };
+      return (
+        <>
+          <Box childGap="md" direction="row">
+            <Button onClick={handleClick}>
+              Open non-dismissible toast
+            </Button>
+          </Box>
+        </>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Toast Types
+
+<Canvas isExpanded>
+  <Story name="Basic Types">
+    {() => {
+      const handleClick = (type) => {
+        if (type === 'success') {
+          const successToastId = toast.success('Proposal saved');
+        } else if (type === 'error') {
+          toast.error('Unable to save');
+        } else if (type === 'loading') {
+          const toastId = toast.loading('Loading...', { duration: 5000000, isDismissable: false });
+        } else {
+          toast('Toast without icon');
+        }
+      };
+      return (
+        <>
+          <Box fontSize="lg" fontWeight="bold" margin="0 0 md 0">
+            Basic Types
+          </Box>
+          <Box childGap="md" direction="row">
+            <Button onClick={() => handleClick('blank')} variant="light">
+              blank
+            </Button>
+            <Button onClick={() => handleClick('error')} variant="danger">
+              error
+            </Button>
+            <Button onClick={() => handleClick('success')} variant="success">
+              success
+            </Button>
+            <Button onClick={() => handleClick('loading')} variant="dark">
+              loading
+            </Button>
+          </Box>
+        </>
+      );
+    }}
+  </Story>
+</Canvas>
+
+<Canvas isExpanded>
+  <Story name="Advanced Types">
+    {() => {
+      const handleClick = (type) => {
+        if (type.includes('async')) {
+          const promise = new Promise((resolve, reject) => {
+            const randomNum = Math.floor(Math.random() * 2)
+            setTimeout(() => {
+              if (randomNum % 2 === 0) {
+                resolve('this is your data');
+              } else {
+                reject('bad request!');
+              }
+            }, 2000);
+          });
+          toast.async(promise, {
```

Comment:
For the async promise scenario, shouldn't `isDismissable` be set to false since we're relying on the promise to set when the toast disappears?

Reply:
yes, this makes sense!